spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+RSpec.describe BooksController do
+  describe '#index' do
+    let!(:books) { create_list(:book, 3) }
```

Comment:
Minimize the count of objects
```suggestion
    let!(:books) { create_list(:book, 2) }
```